openedx/core/djangoapps/programs/tests/test_views.py

```diff
@@ -0,0 +1,98 @@
+"""
+Tests for the Programs.
+"""
+
+from mock import patch
+
+from provider.oauth2.models import Client
+
+from openedx.core.djangoapps.programs.views import get_course_programs_for_dashboard
+from openedx.core.djangoapps.programs.tests.test_models import ProgramsApiBasicTestConfig
+from student.tests.factories import AnonymousUserFactory, UserFactory
+
+
+class TestGetXSeriesPrograms(ProgramsApiBasicTestConfig):
+    """
+    Tests for the Programs views.
+    """
+
+    def setUp(self, **kwargs):  # pylint: disable=unused-argument
+        super(TestGetXSeriesPrograms, self).setUp()
+        self.create_config(enabled=True, enable_student_dashboard=True)
+        Client.objects.get_or_create(name="programs", client_type=0)
```

Comment:
I found it: https://github.com/caffeinehit/django-oauth2-provider/blob/master/provider/constants.py#L7
Its a tuple. So I think providing it like this is fine.